cmd/rotate_certs.go

```diff
@@ -359,7 +373,7 @@ func (rcc *rotateCertsCmd) getKubeClient() (armhelpers.KubernetesClient, error)
 }
 
 // Rotate etcd CA and certificates in all of the master nodes.
-func (rcc *rotateCertsCmd) rotateEtcd(ctx context.Context) error {
+func (rcc *rotateCertsCmd) rotateEtcd() error {
```

Comment:
@jackfrancis or @CecileRobertMichon any ideas on why we don't use `scp` rather than ssh'ing into the machine and executing the bash script?

Reply:
I don't remember a good reason, but possibly because the nodes don't have a public IP so we need to SSH to the master first to be able to access the nodes